src/App.js

```diff
@@ -2,22 +2,35 @@ import React, { useEffect, useState } from "react";
 import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
 
+
 function App() {
-  const [todoList, setTodoList] = useSemiPersistentState();
+  const [todoList, setTodoList] = useState([]); //updated to empty array
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() =>{
+    const myPromise = new Promise((resolve, reject) => {
```

Comment:
Nitpick:
You don't need to assign the `new Promise` to a variable named `myPromise`. Instead you can just define the promise without a variable and chain `.then` afterward:
```js
new Promise(...etc).then(...etc)
```